### PR TITLE
ECS-3077 Apply style guide, tighten some sentences.

### PR DESCRIPTION
**Pull request by NeilBurgess42**
## Content

This PR includes a review of the protocol description in protocol.md

## Pre-submit checklist

- Branch
  - [x] Tests are provided (if possible)
  - [x] Crates versions are updated (if relevant)
  - [x] Commit sequence broadly makes sense
  - [x] Key commits have useful messages
- PR
  - [x] No clippy warnings in the CI
  - [x] Self-reviewed the diff
  - [x] Useful pull request description
  - [x] Reviewer requested
- Documentation
  - [x] Update README file (if relevant)
  - [x] Update documentation website (if relevant)
  - [x] Add dev blog post (if relevant)

## Issue(s)
Relates to ECS-3077
Relates to #YYY or Closes #YYY

### CODE REVIEW

**Comment by olgahryniuk** [comment]
Committed some more improvements. Looks good, @NeilBurgess42 please mark the PR as 'ready for review'.
Over to @jpraynaud .
Thanks so much, everyone!

File: docs/root/mithril/mithril-protocol/protocol.md [hunk 1]
```diff
@@ -3,126 +3,119 @@ sidebar_position: 1
...
 
-Note that all 3 phases require the set of parties (`P` in the paper) to be fixed. During the protocol establishment phase
+Note that all three phases require a set of parties (`P` in the paper) to be fixed. During the protocol establishment phase
 three important parameters are generated:
 
-* `m`, defines the number of "lotteries" that a single user can participate in to sign a message.
-* `k`, defines the required number of single signatures to produce a valid certificate.
-* `phi_f` (as denoted in the library) which can be interpreted as the chance of a signer to win a lottery.
+* `m`: defines the number of lotteries that a single user can participate in to sign a message
+* `k`: defines the required number of signatures for a valid certificate
```
Comment (jpraynaud) [inline docs/root/mithril/mithril-protocol/protocol.md:53]: Is it still accurate as we send all the won lotteries indices along with one message signature now @iquerejeta @curiecrypt ?

Comment (iquerejeta) [reply docs/root/mithril/mithril-protocol/protocol.md:53]: Yes, it seems to me that it is still correct. Even if we send more than `k` signatures in the certificate, that is still the security parameter that defines the minimal number of signatures. i.e. at least `k` signatures are need to generate a valid certificate.

File: docs/root/mithril/mithril-protocol/protocol.md [hunk 2]
```diff
@@ -3,126 +3,119 @@ sidebar_position: 1
...
-  idealised" manner described in the paper. Instead, all signers will register against all other signers. Meaning that
-  each signer will trace it's own registration procedure. Together with other participants' keys, the stake of each
-  party is stored.
-* Then an aggregate verification key (`AVK` in the paper) is created from the registration material, in the form of a
-  Merkle-tree.
+Firstly, the party retrieves the protocol parameters.  
 
-The key dissemination process can also happen on-chain, which makes sense as the parties need to have some guarantees about the validity of keys and stakes of each other party.
+Next, they use the parameters to generate a new key pair of a verification key and a signing key, plus a proof of possession (PoP)
+  `κ`. The verification key and PoP are broadcast to all other parties for registration, which is expected to last for a limited period. An important difference between the implementation and the paper is that registration is not centralized in the 'idealized' manner described in the paper. Instead, all signers will register against all other signers. Thus
```
Comment (iquerejeta) [inline docs/root/mithril/mithril-protocol/protocol.md:86]: Here it says that registration is not centralised. Maybe we want to change that? @jpraynaud

**Review by iquerejeta — APPROVED** [review]
Minor comments. Otherwise, LGTM

File: docs/root/mithril/mithril-protocol/protocol.md (hunk 2 above)
Comment (jpraynaud) [reply docs/root/mithril/mithril-protocol/protocol.md:86]: It looks OK, as it says that the registration is ideally not centralized, but is centralized in the current implementation IIUC. WDYT @iquerejeta?

Comment (iquerejeta) [reply docs/root/mithril/mithril-protocol/protocol.md:86]: Oh, you are right. I misread. It says that a difference is that in the paper registration is not centralised. Sorry, my bad 🙏  no further comments.

Comment (jpraynaud) [reply docs/root/mithril/mithril-protocol/protocol.md:86]: No problem, you made me doubt :sweat_smile:

**Review by curiecrypt — APPROVED** [review]
LGTM 💯

**Pull request merged**